addons/mod_tool/utils.gd

```diff
@@ -55,6 +56,38 @@ static func output_error(message: String) -> void:
 	ModToolStore.label_output.append_bbcode("\n [color=%s]ERROR: %s[/color]" % [ModToolStore.error_color, message])
 
 
+static func output_info(message: String) -> void:
+	ModToolStore.label_output.append_bbcode("\n" + message)
+
+
+# Validates the mod directory name based on the rules defined in the ModLoader ModManifest Class.
+static func validate_mod_dir_name(dir_name: String) -> bool:
```

Comment:
this should be the same as `ModManifest.is_mod_id_valid` right?

Reply:
How did I miss the existence of that one 👀 

That would be something along this lines - right?
`ModManifest.is_mod_id_valid(ModToolStore.name_mod_dir, ModToolStore.name_mod_dir, "", true)`

Reply:
fixed [d5bacda](https://github.com/GodotModding/godot-mod-tool/pull/24/commits/d5bacda1af36d1b27b60defa6c54ccc20f4c1466) 👍